packages/components/src/templates/forms.stories.ts

```diff
@@ -0,0 +1,70 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+/**
+ * ```
+ * ```
+ */
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Forms',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3060-14503&t=LHCyCYMjDV22qemZ-4'
+    }
+  }
+};
+
+export const loginForm = {
+  render: () => html`
+    <section class="sd-container">
+      <form>
+        <h3 class="sd-headline sd-headline--size-3xl mb-8">Login for UnionFondsOnline users</h3>
+        <sd-input type="text" label="Access number" spellcheck class="mb-4"></sd-input>
+        <sd-input type="password" label="Access number" password-toggle spellcheck class="mb-4"></sd-input>
+        <sd-checkbox>Stay logged in on this computer</sd-checkbox>
+        <div class="justify-between md:items-center md:flex-row mt-8 flex flex-col">
+          <sd-link href="#" class="mb-4 md:mb-0">Forgot password</sd-link>
+          <sd-button>
+            <sd-icon name="system/lock-locked" slot="icon-left"></sd-icon>
+            Login
+          </sd-button>
+        </div>
+      </form>
+    </section>
+  `
+};
+
+export const contactForm = {
+  render: () => html`
+    <form>
+      <h3 class="sd-headline sd-headline--size-4xl mb-8">Contact</h3>
+      <p class="sd-paragraph sd-paragraph--size-sm mb-8">Please fill out all fields marked *.</p>
```

Comment:
```suggestion
      <p class="sd-paragraph sd-paragraph--size-sm mb-8">Please fill out all fields marked *.</p>
```

[Here](https://www.union-investment.de/unsere-services/news-abonnements/anleger-newsletter/newsletter-abonnieren?email=) it says "Pflichtfeld", which is more or less "Required fields" or "Required". Is this our recommendation from UX perspective?

Or should it even be at the bottom and as a meta-text?

Reply:
Updated in code as well